elide-model-config/src/main/java/com/yahoo/elide/modelconfig/jsonformats/ElideNamespaceNameFormatAttr.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.modelconfig.jsonformats;
+
+import com.github.fge.jackson.NodeType;
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
+import com.github.fge.jsonschema.core.report.ProcessingReport;
+import com.github.fge.jsonschema.format.AbstractFormatAttribute;
+import com.github.fge.jsonschema.processors.data.FullData;
+import com.github.fge.msgsimple.bundle.MessageBundle;
+
+import java.util.Locale;
+import java.util.regex.Pattern;
+
+/**
+ * Format specifier for {@code elideNamespaceName} format attribute.
+ * <p>
+ * This specifier will check if a string instance is a valid Elide Name.
+ * </p>
+ */
+public class ElideNamespaceNameFormatAttr extends AbstractFormatAttribute {
+    public static final Pattern NAME_FORMAT_REGEX = ElideNameFormatAttr.NAME_FORMAT_REGEX;
+
+    public static final String FORMAT_NAME = "elideNamespaceName";
+    public static final String FORMAT_KEY = "elideNamespaceName.error.format";
+    public static final String FORMAT_MSG =
+                    "Name [%s] is not allowed. Name must start with an alphabetic character and can include "
+                    + "alaphabets, numbers and '_' only. Name must not be a variant of 'default' with different cases.";
```

Comment:
Can these be two different messages for the two different error scenarios?  Also, the second condition is a little hard to follow.  Maybe say something like:  'Name [%s] clashes with the 'default' namespace.  Either change the case or pick a different namespace name.'